app/org/sagebionetworks/bridge/services/StudyServiceImpl.java

```diff
@@ -39,33 +41,59 @@ public Study load(String identifier) throws EntityNotFoundException {
     private StudyValidator validator;
     private CacheProvider cacheProvider;
     private StudyConsentService studyConsentService;
+
+    private StudyConsentForm defaultConsentDocument;
+    private String defaultEmailVerificationTemplate;
+    private String defaultEmailVerificationTemplateSubject;
+    private String defaultResetPasswordTemplate;
+    private String defaultResetPasswordTemplateSubject;
     
+    @Value("classpath:study-defaults/consent.xhtml")
+    void setDefaultConsentDocument(org.springframework.core.io.Resource resource) {
+        this.defaultConsentDocument = new StudyConsentForm(BridgeUtils.toStringQuietly(resource));
+    }
+    @Value("classpath:study-defaults/email-verification.txt")
+    void setDefaultEmailVerificationTemplate(org.springframework.core.io.Resource resource) {
+        this.defaultEmailVerificationTemplate = BridgeUtils.toStringQuietly(resource);
+    }
+    @Value("classpath:study-defaults/email-verification-subject.txt")
+    void setDefaultEmailVerificationTemplateSubject(org.springframework.core.io.Resource resource) {
+        this.defaultEmailVerificationTemplateSubject = BridgeUtils.toStringQuietly(resource);
+    }
+    @Value("classpath:study-defaults/reset-password.txt")
+    void setDefaultPasswordTemplate(org.springframework.core.io.Resource resource) {
+        this.defaultResetPasswordTemplate = BridgeUtils.toStringQuietly(resource);
+    }
+    @Value("classpath:study-defaults/reset-password-subject.txt")
+    void setDefaultPasswordTemplateSubject(org.springframework.core.io.Resource resource) {
+        this.defaultResetPasswordTemplateSubject = BridgeUtils.toStringQuietly(resource);
+    }
     @Resource(name="uploadCertificateService")
-    public void setUploadCertificateService(UploadCertificateService uploadCertService) {
+    void setUploadCertificateService(UploadCertificateService uploadCertService) {
```

Comment:
As long as we're changing it, might as well make it "final void". While it's unlikely someone will ever subclass StudyServiceImpl, if they do, we don't want them to override the Spring setters.